.github/actions/sign/action.yml

```diff
@@ -0,0 +1,115 @@
+  APPLE_DEV_APP_ID:
+    required: true
+    description: Apple Certificate Developer Name and Team. Eg. Name Surname (1ABCD23EFG)
+runs:
+  using: composite
+  steps:
+    - name: Sign
+      shell: pwsh
+      run: |
+        Set-StrictMode -Version Latest
+        $ErrorActionPreference = "Stop"
+
+        $CodesignPath = Get-Command codesign | Resolve-Path
+
+        $CertificateBase64 = "${{inputs.APPLE_CERT_BASE64}}"
+        $CertificatePassword = "${{inputs.APPLE_CERT_PASSWORD}}"
+        $CertificatePath = [IO.Path]::ChangeExtension((New-TemporaryFile), "p12")
+
+        $Keychain = "ephemeral.keychain"
+        $KeychainPassword = (New-Guid).ToString().Replace("-", "")
+
+        $DevId = "${{ inputs.APPLE_DEV_ID }}"
+        $DevTeamId = "${{ inputs.APPLE_DEV_TEAM_ID }}"
+        $DevPassword = "${{ inputs.APPLE_DEV_PASSWORD }}"
+        $DeveloperIdApplication = "${{ inputs.APPLE_DEV_APP_ID }}"
+
+        if (!$CertificateBase64) { throw "No certificate provided" }
+        if (!$CertificatePassword) { throw "No certificate password provided" }
+        if (!$DevId) { throw "No Apple Developer ID provided" }
+        if (!$DeveloperIdApplication) { throw "No Apple Developer ID Application provided" }
+        if (!$DevTeamId) { throw "No Apple Team ID provided" }
+        if (!$DevPassword) { throw "No Apple Developer password provided" }
+
+        Write-Output "Decoding certificate..."
+
+        $Certificate = [Convert]::FromBase64String($CertificateBase64)
+
+        Write-Output "Writing certificate to disk..."
+
+        [IO.File]::WriteAllBytes($CertificatePath, $Certificate)
+
+        Write-Output "Creating keychain..."
+
+        security create-keychain -p $KeychainPassword $Keychain
+
+        Write-Output "Setting keychain as default..."
+
+        security default-keychain -s $Keychain
+
+        Write-Output "Importing certificate into keychain..."
+
+        security import $CertificatePath `
+          -k ~/Library/Keychains/$Keychain `
+          -P $CertificatePassword `
+          -T $CodesignPath
+
+        Write-Output "Granting access to keychain..."
+
+        security set-key-partition-list -S "apple-tool:,apple:" -s -k $KeychainPassword $Keychain
+
+        $Framework = "${{ inputs.FRAMEWORK_PATH }}"
+        $Archive = [IO.Path]::ChangeExtension((New-TemporaryFile), "zip")
+
+        Write-Output "Signing '$Framework'..."
+
+        & $CodesignPath --force --verify --timestamp --verbose --sign $DeveloperIdApplication "$Framework"
+
+        Write-Output "Verifying signing..."
+
+        & $CodesignPath --verify --verbose "$Framework"
+
+        Write-Output "Archiving framework to '$Archive'..."
+
+        ditto -ck --sequesterRsrc --keepParent "$Framework" "$Archive"
+
+        Write-Output "Submitting archive for notarization..."
+
+        $output = xcrun notarytool submit "$Archive" `
+          --apple-id $DevId `
+          --team-id $DevTeamId `
+          --password $DevPassword `
+          --wait
```

Comment:
I'm curious why you decided to omit the MIT license attribution of the [original script](https://github.com/godot-jolt/godot-jolt/blob/master/scripts/ci_sign_macos.ps1), given that you correctly attributed it in the four other repositories ([1](https://github.com/appsinacup/godot-box2d/blob/f717f7534da19fa2618b82fbcdafc040093aaafe/THIRDPARTY.txt#L74-L93), [2](https://github.com/appsinacup/godot-rapier-2d/blob/3fbb73ba88e3ecf6481be1a23a410eba9e119c92/THIRDPARTY.txt#L66-L85), [3](https://github.com/V-Sekai/godot-whisper/blob/7ba3bb4dc9d88a1f3d7db41b4b1fed19fd9a2895/THIRDPARTY.txt#L3-L22), [4](https://github.com/V-Sekai/godot-llama/blob/eabc2a1252ff540ff71517e46c0997c988422811/THIRDPARTY.txt#L3-L22)) that you copied this into.

Reply:
Ups 🙃 
Sorry, a lot of copy pasting around. Will update.

Reply:
Btw, should also update thirdparty.txt for all repos and this with the link to jolt repo as well. Will update that too.